packages/api/tests/lib/test-granule-remove-from-cmr.js

```diff
@@ -150,22 +137,13 @@ test.serial('unpublishGranule() succeeds with Dynamo and PG granule', async (t)
   const { fakeCollection } = t.context;
```

Comment:
Looks like we need to rename this test? `unpublishGranule() succeeds with PG granule`

Reply:
67cd4e8faf